frontend/src/app/main/data/workspace/undo.cljs

```diff
@@ -170,9 +169,9 @@
   (ptk/reify ::check-open-transactions
     ptk/WatchEvent
     (watch [_ state _]
-      (log/info :msg "check-open-transactions")
+      (log/info :hint "check-open-transactions")
       (let [pending-ts (-> (dm/get-in state [:workspace-undo :transactions-pending-ts])
-                           (update-vals #(.toMillis (dt/diff (dt/now) %))))]
+                           (update-vals #(inst-ms (dt/diff (dt/now) %))))]
         (->> pending-ts
              (filter (fn [[_ ts]] (>= ts discard-transaction-time-millis)))
```

Comment:
Shouldn't we use here the `timeout` parameter?

If someone gives a valid timeout but that is inferior to `discard-transaction-time-ms` the transactions will not be forced at this point.

Reply:
you are right, i'm going to fix it

Reply:
perfect